### ISSUE TITLE
async howto implies Deferreds solve concurrency issues

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2719]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#2719|
|Type|defect|
|Created|2007-06-25 11:58:17Z|
|Branch|https://github.com/twisted/twisted/tree/remove-async-howto-2719|

```
It is the second class of concurrency problem - non-computationally intensive tasks that involve an appreciable delay - that Deferreds are designed to help solve. Functions that wait on hard drive access, database access, and network
access all fall into this class, although the time delay varies.
```

...

```
The basic idea behind Deferreds, and other solutions to this problem, is to keep the CPU as active as possible.
```

I think this howto is somewhat misleading and sorta implies that Deferreds have something to do with Twisted's concurrency model, a confusion many new users have.

<details><summary>Searchable metadata</summary>

```
trac-id__2719 2719
type__defect defect
reporter__itamarst itamarst
priority__normal normal
milestone__ 
branch__branches_remove_async_howto_2719 branches/remove-async-howto-2719
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1182772697000000 1182772697000000
changetime__1248908819000000 1248908819000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

I think we should just delete this howto entirely.  It looks like a bad re-hash of the concepts covered in "Deferred Reference".

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Replying to [glyph](#note_1):
> I think we should just delete this howto entirely.  It looks like a bad re-hash of the concepts covered in "Deferred Reference".

Anyone want to vote on this? It's gone before you know it ;)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [27101]) Branching to 'remove-async-howto-2719'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Removed the document and moved a part of it into the Deferred howto. Up for review because the [doc builder](http://buildbot.twistedmatrix.com/builders/documentation/builds/524) is happy.

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_5"></a>|@jonathanj set owner to @thijstriemstra|
|-|-|

This definitely looks like an improvement to me. I don't see any errors in the new content, so I'd say go ahead and merge this.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27176]) Merge remove-async-howto-2719: Remove async howto and references in other docs
because it's a bad re-hash of the concepts covered in the Deferred Reference
howto.

Author: thijs
Reviewer: glyph, jonathanj
Fixes: [#2719](https://github.com/twisted/twisted/issues/2719)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

Note that while the merge commit indicates Glyph was a reviewer on this ticket, he wasn't.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
